x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -457,14 +446,25 @@ export const signalRulesAlertType = ({
           if (query === undefined) {
             throw new Error('EQL query rule must have a query defined');
           }
-          const MIN_EQL_RULE_TEMPLATE_VERSION = 2;
-          if (
-            signalMappingVersion === undefined ||
-            signalMappingVersion < MIN_EQL_RULE_TEMPLATE_VERSION
-          ) {
-            throw new Error(
-              `EQL based rules require an update to version ${MIN_EQL_RULE_TEMPLATE_VERSION} of the detection alerts index mapping`
-            );
+          const MIN_EQL_RULE_INDEX_VERSION = 2;
```

Comment:
I think this should be placed next to https://github.com/elastic/kibana/pull/81277/files#diff-efd661b260b883b4c79fd6836bc484534c6cc4f5c40291812bcfff599000e070R10 so that versions can be directly compared side by side.